lib/src/fields/form_builder_dropdown.dart

```diff
@@ -253,6 +253,7 @@ class FormBuilderDropdown<T> extends FormBuilderFieldDecoration<T> {
     this.enableFeedback,
     this.borderRadius,
     this.alignment = AlignmentDirectional.centerStart,
+    Widget? hint,
```

Comment:
Would be a parameter defined like with this.hint, so need to declare like final above, like others

Reply:
Done

Reply:
@deandreamatias